pkg/sql/ttl/ttljob/ttljob_test.go

```diff
@@ -46,6 +47,8 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+var zeroDuration time.Duration
```

Comment:
wonder if you can make this `const` - `const zeroDuration = time.Duration(0)`

Reply:
It looks like that doesn't help since we would need to assign the `const` to a `var` to get its address.

https://stackoverflow.com/questions/35146286/find-address-of-constant-in-go